examples/e2ee/main.mjs

```diff
@@ -0,0 +1,126 @@
+// Description:
+//
+// Example showing how to send an end-to-end-encrypted (E2EE) message via
+// Matrix.
+
+// Requirements:
+//
+// - A user must exist (enter credentials below).
+// - A room must exist, where the existing user is a joined member (enter room
+//   ID, below).
+//
+// Tested with:
+//
+// - NodeJS v18.17.1
+// - matrix-js-sdk v28.2.0
+// - olm v3.1.5
+
+import * as sdk from 'matrix-js-sdk';
+import olm from 'olm';
+
+const sLogPrefix = '*** MAIN ***: ',
+    sC = {
+        userName: '********',
+        password: '********',
+        rawUrl: 'matrix.org',
+
+        roomId: '********',
+        msg: 'Hello World!'
+    };
+
+function log(msg)
+{
+    console.log(`${sLogPrefix}${msg}`);
+};

```

Comment:
This whole log wrapper and prefix feels weird and distracting for an example, especially since the prefix is hardcoded.